### DISCUSSION TITLE
Exercice de maquettes en argile

### DISCUSSION THREAD

**Discussion by sjinko** [CUPUM/ecometropole_laurentienne#39] [General]
Bonjour! Voici une discussion où nous pouvons discuter de l'exercice des maquettes en argile. Les messages laissés ici ne sont pas envoyés par courriel, sauf si vous mentionnez quelqu'un directement en utilisant l'identifiant GitHub (p.ex. Le miens est @sjinko).

**Comment by sjinko** [comment]
@Sarahprairie, si tu as des questions, tu peux les poser ici!

**Reply by Sarahprairie** [reply]
Est-ce que c'est possible d'utiliser des fils de fer ou d'autres matériaux pour faire la base et ensuite la recouvrir d'argile (pour que ce soit plus solide) ?

**Reply by sjinko** [reply]
À ce stade, je préfère que vous vous concentriez sur les qualités et limites de l'argile seulement. Nous pourrons parler d'autres matériaux lundi.
